esports_data_from_rimble/README.md

```diff
@@ -0,0 +1,339 @@
+# Getting Esports Data From Rimble
+
+This example shows you how to use the Blocky Attestation Service (Blocky AS) to
+attest a function call that fetches data from the Rimble API and processes
+it.
+
+Before starting this example, make sure you are familiar with the
+[Hello World - Attesting a Function Call](../hello_world_attest_fn_call/README.md),
+[Passing Input Parameters and Secrets](../params_and_secrets/README.md),
+and [Error Handling](../error_handling/README.md) examples.
+
+In this example, you'll learn how to:
+
+- Fetch data from the Rimble API.
+
+## Setup
+
+- Install the Blocky AS CLI by following the
+  [setup instructions](https://blocky-docs.redocly.app/v0.1.0-beta.6/attestation-service/setup)
```

Comment:
Please, make sure that the version in the urls are updated to be templated.  See https://github.com/blocky/attestation-service-examples/pull/35

Reply:
done